azure/azure-how-to/instances/launch-ubuntu-images.rst

```diff
@@ -0,0 +1,130 @@
+Launch Ubuntu Images on Azure
```

Comment:
Would it be more appropriate to say 'Launch Ubuntu instances on Azure'? instead of 'Launch Ubuntu images on Azure'? (This is a question that I am asking, not a suggestion.)

Reply:
Images is more consistent with the operation, as "Ubuntu instances" would tie Ubuntu to the hardware. The user is launching Ubuntu images on Azure instances.